### DISCUSSION TITLE
Understanding the response curves

### DISCUSSION THREAD

**Discussion by ArrunAnanda** [facebookexperimental/Robyn#408] [Q&A]
Hi there,

I need some help understanding the response curves. For this question I'm using the dt_simulated_weekly sample data provided by Meta in the package.

Let's TV spending as an example - I want to know from where is the 256k coming from that is pointed on the line? 

1. Is it a transformed value?
2. If it's a transformed value, then is it indicating the spend in the model window which is 92 weeks (2016-11-21 and 2018-08-20) or the whole time period of the data (2015-11-23 and 2018-08-20) which is 144 weeks?

Thanks!

<img width="807" alt="image" src="https://user-images.githubusercontent.com/59400103/173847263-af78347c-9071-4391-822d-80841b7afb87.png">

**Comment by kyletgoldberg** [comment]
Hi - Good questions. The 256k comes from the mean non-zero spend days. 

You can see the code for the underlying data set here - 
![image](https://user-images.githubusercontent.com/84025306/173902369-be0f7080-ef2e-46fa-954a-2dda42d3d640.png)

Notice that the x-values going into the saturation function come from the adstocked spend values. So yes they are transformed

**Comment by ArrunAnanda** [comment]
Thanks for the quick response. Now, I understand. @kyletgoldberg 

What is the timeframe that is considered for mean non-zero spend days? Is it the rolling window **2016-11-21 and 2018-08-20** or the whole time period of the data **2015-11-23 and 2018-08-20**?
